include/resize-frame.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_RESIZE_FRAME_H
+#define LABWC_RESIZE_FRAME_H
+
+#include "view.h"
+#include <wlr/util/box.h>
+
+void resize_frame_update(struct view *view, struct wlr_box new_geo);
+void resize_frame_finish(struct view *view);
```

Comment:
`struct view` could be forward declared rather than including `view.h`

Reply:
Thanks. Fixed.